reformer_pytorch/reformer_pytorch.py

```diff
@@ -148,7 +148,7 @@ def hash_vectors(self, n_buckets, vecs):
 
         if self._rehash_each_round:
             rotated_vecs = torch.cat([rotated_vecs, -rotated_vecs], dim=-1)
-            buckets = torch.argmax(rotated_vecs, axis=-1)
+            buckets = torch.argmax(rotated_vecs, dim=-1)
```

Comment:
Interesting, it works even with `axis`

Reply:
Okay! axis raised an error for me. PyTorch version difference might be the issue. So, adding specific (torch==1.4.0) in requirements.txt will help in 'functional.glu' line  as well.